docs/modules/ROOT/pages/contributors/documentation.adoc

```diff
@@ -0,0 +1,93 @@
+= Documentation Style
+
+When contributing documentation to KubeArchive you are expected to follow
+the guidelines listed below:
+
+* Use present tense, instead of future or past.
+* Use the active voice, instead of the passive voice.
+* Use the second and third persons, instead of first person.
+* Use positive phrases, instead of negative phrases.
+* Use cautious language, instead of making claims.
+* Do not recommend.
+* Use english abbreviations, instead of latin abbreviations.
+
+== Tense
+
+Use present tense whenever possible:
+
+NO: When configured in this way KubeArchive will delete resources.
```

Comment:
This comment is just about preference (another option), I would add the examples as `CORRECT` and `INCORRECT` instead of YES` and `NO`

Reply:
I will keep it as is